codegen/src/main/java/io/spine/tools/proto/FileName.java

```diff
@@ -114,23 +96,61 @@ public String nameOnlyCamelCase() {
      * Returns {@code true} if the name of the file matches convention for command message files.
      */
     public boolean isCommands() {
-        final boolean result = value().endsWith(COMMANDS_FILE_SUFFIX);
+        final boolean result = value().endsWith(Suffix.forCommands());
         return result;
     }
 
     /**
      * Returns {@code true} if the name of the file matches convention for event message files.
      */
     public boolean isEvents() {
-        final boolean result = value().endsWith(EVENTS_FILE_SUFFIX);
+        final boolean result = value().endsWith(Suffix.forEvents());
         return result;
     }
 
     /**
      * Returns {@code true} if the name of the file matches convention for rejection message files.
      */
     public boolean isRejections() {
-        final boolean result = value().endsWith(REJECTIONS_FILE_SUFFIX);
+        final boolean result = value().endsWith(Suffix.forRejections());
         return result;
     }
+
+    /**
+     * Constants for names of standard message files.
+     */
+    public static class Suffix {
+
+        /** The standard file extension. */
+        private static final String EXTENSION = ".proto";
+
+        private static final String FOR_COMMANDS = CommandMessage.File.SUFFIX;
```

Comment:
Maybe. I wanted to avoid capital case for enum members, and make it shorter. Compare:
```java
FileName.Suffix.FOR_COMMANDS.name()
```

```java
FileName.Suffix.forCommands()
```